src/parser/csql_grammar.y

```diff
@@ -24884,49 +24881,35 @@ dblink_column_definition_list
                PARSER_SAVE_ERR_CONTEXT ($$, @$.buffer_pos)
              DBG_PRINT}}
         ;
-    
+
 dblink_column_definition
-        : IdName primitive_type
+        : identifier primitive_type
```

Comment:
`dblink column definition` can't have set types?
There exist similar routines with this. But they have `data_type` instead of `primitive_type`
```
OOO
| identifier data_type
```

Reply:
Except for CLOB, B:OB, and OBJECT types, only primitive_type is supported.